pubpub/pubpub
# feature: Enable unanchored discussions in V6

While there's no technical reason we can't have discussions at the bottom of the Pub unassociated with any highlight, we haven't had this capability at least since v6 launched. This PR adds a comment box at the bottom of the pub to enable this.

_Test plan:_
- Load a pub and make sure you can add (multiple) comments using the bottom discussion box
- Make sure the box does not appear when you don't have discuss permissions. 

![image](https://user-images.githubusercontent.com/2208769/63030335-2194c500-be80-11e9-8fe3-857eacbdfb83.png)
